ext/odbc/php_odbc.c

```diff
@@ -3531,10 +3531,6 @@ PHP_FUNCTION(odbc_procedurecolumns)
 	size_t cat_len = 0, schema_len = 0, proc_len = 0, col_len = 0;
 	RETCODE rc;
 
-	if (ZEND_NUM_ARGS() != 1 && ZEND_NUM_ARGS() != 5) {
```

Comment:
This will change the behavior, as now not only 1 and 5, but also 2, 3, 4 arguments will be allowed.

Reply:
Oh, TIL, I thought we could only allow sequential args, like 1 up to 5, not 1, 3 or 5 :sweat_smile: